strings.js

```diff
@@ -0,0 +1,54 @@
+function reversal(inputText) {
+	var stringArray = inputText.split('');
+	var reverseString = stringArray.reverse().join('');
+	output1.innerHTML = "reversal: " + reverseString;
+}
+
+function alphabits(inputText) {
+	var stringArray = inputText.split('');
+	var alphaString = stringArray.sort().join('');
+	output2.innerHTML = "alphabits: " + alphaString;
+}
+
+function palindrome(inputText) {
+	var stringArray = inputText.split('');
+	var reverseArray = stringArray.reverse().join('');
+	if (inputText===reverseArray) {
+		output3.innerHTML = "Your string is a palindrome";
+	} 
+	else {
+		output3.innerHTML = "";
+	}
+
+}
+
+
+var testString = "";
+var textBox = document.getElementById("uInput");
+var submitButton = document.getElementById("submitInfo");
+var output1 = document.getElementById("reversalOutput");
```

Comment:
I would call this reversedString or something that is very explicit about what it is doing